src/Ilios/CoreBundle/Entity/Repository/LearnerGroupRepository.php

```diff
@@ -95,6 +95,39 @@ public function findDTOsBy(array $criteria, array $orderBy = null, $limit = null
      */
     protected function attachCriteriaToQueryBuilder(QueryBuilder $qb, $criteria, $orderBy, $limit, $offset)
     {
+        if (array_key_exists('cohort', $criteria)) {
```

Comment:
@stopfstedt thoughts on this?  @ctam has existing API calls that look for singular cohort=X so I left in support for that, but we don't have it anywhere else.